bookwyrm/templates/import/import_user.html

```diff
@@ -0,0 +1,156 @@
+{% extends 'layout.html' %}
+{% load i18n %}
+{% load humanize %}
+
+{% block title %}{% trans "Import User" %}{% endblock %}
+
+{% block content %}
+<div class="block">
+    <h1 class="title">{% trans "Import User" %}</h1>
+
+    {% if invalid %}
+    <div class="notification is-danger">
+        {% trans "Not a valid import file" %}
+    </div>
+    {% endif %}
+
+    {% if not site.imports_enabled %}
+    <div class="box notification has-text-centered is-warning m-6 content">
+        <p class="mt-5">
+            <span class="icon icon-warning is-size-2" aria-hidden="true"></span>
+        </p>
+        <p class="mb-5">
+            {% trans "Imports are temporarily disabled; thank you for your patience." %}
+        </p>
+    </div>
+    {% elif next_available %}
+        <div class="notification is-warning">
+            <p>{% blocktrans %}Currently you are allowed to import one user every {{ user_import_hours }} hours.{% endblocktrans %}</p>
+            <p>{% blocktrans %}You will next be able to import a user file at {{ next_available }}{% endblocktrans %}</p>
+        </div>
+    {% else %}
+    <form class="box" name="import-user" action="/user-import" method="post" enctype="multipart/form-data">
+        {% csrf_token %}
+
+        <div class="columns">
+        <div class="column is-half">
+            <div class="field">
+                <label class="label" for="id_archive_file">{% trans "Data file:" %}</label>
+                {{ import_form.archive_file }}
+            </div>
+            <div>
+                <p class="block"> {% trans "Importing this file will overwrite any data you currently have saved." %}</p>
```

Comment:
I think this needs some clarification -- it looks to me like this adds additional data, and _can_ change or override existing data in surprising ways, but doesn't, for example, delete a readthrough that already exists if there's a new one. The wording as is makes it sounds like your current data would be completely cleared (which some people really really want and would be sad when it didn't happen)

Reply:
Yes. Actually I'd really like to go over that with you to make sure we're clear on what we're overwriting and what we're adding (and that what's happening is intentional). It would be good to make it less surprising.

What it _should_ do, I believe, is:

* always overwrite relevant user profile and settings values
* **not** create new editions or authors if there is a matching one
* **not** overwrite existing identical readthroughs, goals, statuses, or lists
* only follow or block users that aren't currently followed or blocked

So the intention is that most things are "no-op" on matching things that already exist, except user settings and profile. My thinking on this is that people might want to migrate in to an account that they've been running as an alt, and don't want to wipe out everything they did previously. Bulk-deleting is really a separate question.

Reply:
@mouse-reeve I see what you mean now and updated this, but it may be too verbose 😬

Reply:
Do you think it would work to combine the category-by-category description in with the checkboxes? That might make it easier to parse